test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeGenerator.java

```diff
@@ -551,21 +547,38 @@ private static void generateGetter(ClassWriter cw, String[] nameAndSigValue, Str
 		default:
 			break;
 		}
+
 		mv.visitInsn(ARETURN);
 		mv.visitMaxs(2, 1);
 		mv.visitEnd();
 	}
-	
-	private static Class<?> defineClass(String className, byte[] bytes, ClassLoader loader, ProtectionDomain pD) throws Throwable {
-		return unsafe.defineClass(className, bytes, 0, bytes.length, loader, pD);
-	}
-	
+
 	public static Class<?> generateValueClass(String name, String[] fields) throws Throwable {
-		return ValueTypeGenerator.defineClass(name, ValueTypeGenerator.generateValue(name, fields), ValueTypeGenerator.class.getClassLoader(), ValueTypeGenerator.class.getProtectionDomain());
+		byte[] bytes = generateValue(name, fields, false);
+		writeBytesToFile(bytes, name);
```

Comment:
Since youll need to change the code to turn this on, I'd prefer if this was removed and added explicitly when debugging is required